concent_api/conductor/tasks.py

```diff
@@ -2,13 +2,60 @@
 
 from celery import shared_task
 
+from .models import BlenderSubtaskDefinition
+from .models import UploadReport
+from .models import VerificationRequest
+
 
 @shared_task
 def blender_verification_request(
-    subtask_id:             str,  # pylint: disable=unused-argument
-    source_package_path:    str,  # pylint: disable=unused-argument
-    result_package_path:    str,  # pylint: disable=unused-argument
-    output_format:          Enum,  # pylint: disable=unused-argument
-    scene_file:             str,  # pylint: disable=unused-argument
+    subtask_id:             str,
+    source_package_path:    str,
+    result_package_path:    str,
+    output_format:          Enum,
+    scene_file:             str,
 ):
-    pass
+    # The app creates a new instance of VerificationRequest in the database
+    # and a BlenderSubtaskDefinition instance associated with it.
+    verification_request = VerificationRequest(
+        subtask_id=subtask_id,
+        source_package_path=source_package_path,
+        result_package_path=result_package_path,
+    )
+    verification_request.full_clean()
+    verification_request.save()
+
+    blender_subtask_definition = BlenderSubtaskDefinition(
+        verification_request=verification_request,
+        output_format=output_format,
+        scene_file=scene_file,
+    )
+    blender_subtask_definition.full_clean()
+    blender_subtask_definition.save()
+
+    # If there are already UploadReports corresponding to some files, the app links them with the VerificationRequest
+    # by setting the value of the foreign key in UploadReport.
+    for path in [source_package_path, result_package_path]:
```

Comment:
Should we check for both `source_package_path` and `source_package_path`?

Reply:
I don't understand the question. Do you mean `source_package_path and result_package_path` ? If yes then yes, we should check both according to issue description.